metaflow/plugins/kfp/kfp.py

```diff
@@ -213,46 +222,41 @@ def create_kfp_pipeline_yaml(self, pipeline_file_path) -> str:
         )
         return os.path.abspath(pipeline_file_path)
 
-    def _command(
-        self,
-        code_package_url: str,
-        environment: MetaflowEnvironment,
-        step_name: str,
-        step_cli: List[str],
-        resource_requirements: Dict[str, str],
-        task_id: str,
-    ) -> str:
-        """
-        Analogous to batch.py
-        """
-        retry_count_python = (
-            "import os;"
-            'name = os.environ.get("MF_ARGO_NODE_NAME");'
-            'index = name.rfind("(");'
-            'retry_count = (0 if index == -1 else name[index + 1: -1]) if name.endswith(")") else 0;'
-            "print(str(retry_count))"
-        )
+    def _cd_into_metaflow_package_cmd(self) -> str:
+        if self.s3_code_package:
+            cd_cmd = ""
+        else:
+            cd_cmd = " && cd " + str(
```

Comment:
Can we have `&&` pushed to outer scope, so that this function is more "dry" in the sense that it is not aware of a command before this. 

For s3 code package you can probably write `"true"` instead of `""`